src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -1596,7 +1608,7 @@ def generate_snapshot_batch(
             msg += f"\n{str(e)}."
             self._raise_error(msg)
 
-    def generate_points(self, named_selection: str, rom_name: str, on_disk: bool = True):
+    def generate_points(self, rom_name: str, on_disk: bool = True, named_selection: str = None):
```

Comment:
Just FYI, we made the `named_selection` argument non-optional because if not there is some redundancy with the `get_geometry_filepath` method. But since the `generate_points` method may returns the points in memory, I'm fine with making the `named_selection` argument optional.

Reply:
I think it's useful to have the `generate_points` method output a file in a similar manner if using `named_selection` or not. We have a workflow where we are mapping the field result onto an FE model, so need both snapshot and points. The downstream workflow is simpler if it can expect a pair with similar names, rather than looking for points elsewhere. Additionally, the calling script may or may not have at a named selection.

As an alternative, to avoid some redundancy, would it be better to just copy (and rename) the points.bin in the case that we call it without `named_selection` ?